VideoLocker/src/main/java/org/edx/mobile/http/Api.java

```diff
@@ -223,6 +224,18 @@ public ProfileModel getProfile() throws Exception {
         // hold the json string as it is
         res.json = json;
 
+
+        // store profile json
+        if (json != null ) {
```

Comment:
Is there a better place to put this? It seems like the API layer shouldn't really be responsible for doing things like updating the database.